backend/compact-connect/lambdas/cloudfront-csp/index.js

```diff
@@ -1,5 +1,337 @@
-// Placeholder for future UI Content Security Policy settings
+//
+//  index.js
+//  CompactConnect
+//
+//  Created by InspiringApps on 7/22/2024.
+//
+
+// ============================================================================
+//                               CONFIGURATION                                =
+// ============================================================================
+/**
+ * Configuration of supported domains per environment.
+ * @type {object}
+ */
+const environments = {
+    csg: {
+        prod: {
+            webFrontend: `app.compactconnect.org`,
+            dataApi: `api.compactconnect.org`,
+            s3Upload: ``, // @TODO: Waiting for environment configuration.
```

Comment:
WIll this be addressed for this PR, or is it a backlog item? If the latter, can we make sure it's being tracked elsewhere besides the code?

Reply:
It'll require the step of setting up the Prod environment. Browsing the board, I didn't see something exactly for that. Do you or @jusdino know if that's part of an existing ticket?

Reply:
Created #119 to track.